app/src/main/java/com/vultisig/wallet/ui/models/deposit/DepositFormViewModel.kt

```
@@ -1165,7 +1168,7 @@ constructor(
             }
         } catch (ce: CancellationException) {
             throw ce
-        } catch (_: Exception) {
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)